src/Compilers/CSharp/Portable/Declarations/MergedTypeDeclaration.cs

```diff
@@ -116,13 +116,13 @@ public int Arity
             }
         }
 
-        public bool ContainsExtensionMethods
+        public bool ContainsExtensionMethodsOrExtensionDeclarations
```

Comment:
>ContainsExtensionMethodsOrExtensionDeclarations [](http://example.com/codeflow?start=20&length=47)

nit: consider simplifying to "ContainsExtensions"

Reply:
No longer applicable